README.md

```diff
@@ -27,39 +33,159 @@
   </a>
 </p>
 
-## Backstory
-
-...
-
 ## Key features
 
 - 🍱 Reliable serialization and deserialization
 - 🔐 Type safety with autocompletion
 - 🐾 Negligible runtime footprint
 - 💫 Framework agnostic
 
-## Example
+
+## Backstory
+
+At [Blitz](https://github.com/blitz-js/blitz), we have struggled with the limitations of JSON. We often find ourselves working with `Date`, `Map`, `Set` or `BigInt`, but `JSON.stringify` doesn't support any of them without going through the hassle of converting manually!
+
+Superjson solves these issues by providing a thin wrapper over `JSON.stringify` and `JSON.parse`.
+
+## Getting started
+
+Install the library with your package manager of choice, e.g.:
+
+```
+yarn add superjson
+```
+
+## Usage
+
+The easiest way to use Superjson is with its `stringify` and `parse` functions. If you know how to use `JSON.stringify`, you already know Superjson!
+
+Easily stringify any expression you’d like:
 
 ```js
-const input = {
+import superjson from 'superjson';
+
+const jsonString = superjson.stringify({date: new Date(0)});
+```
+
+And parse your JSON like so:
+
+```js
+const object = superjson.parse(jsonString);
+```
+
+Alternatively, transform any JavaScript value into a JSON-compatible one by using our lower-level `serializer` and `deserializer` functions.
```

Comment:
I think this should be `serialize` and `deserialize` cause these are the proper names for this functions.